Etabs_Adapter/CRUD/Read/Mesh.cs

```diff
@@ -97,13 +100,7 @@ private List<FEMesh> ReadMesh(List<string> ids = null)
                         //Check if node already has been pulled
                         if (!nodes.TryGetValue(nodeId, out node))
                         {
-                            //Extract node. TODO: to be generalised using read node method
-                            double x = 0;
-                            double y = 0;
-                            double z = 0;
-                            m_model.PointElm.GetCoordCartesian(nodeId, ref x, ref y, ref z);
-                            node = new Node() { Position = new Point { X = x, Y = y, Z = z } };
-                            node.CustomData[AdapterIdName] = nodeId;
+                            node = ReadNode(new List<string>() { nodeId }).FirstOrDefault();
```

Comment:
Dont think we can make this change. Reading nodes is only reading `PointObj`, while reading meshes require reading `PointElm` so would revert back this change